### PR TITLE
Update sbt to 1.3.2

### PR DESCRIPTION
**Pull request by raboof**
Refs #26934 

Draft, to be un-drafted when 1.3.0 is out and various issues are fixed:
* [x] lightbend/sbt-whitesource#64
* [x] https://github.com/lightbend/paradox/issues/311 paradox doesn't appear to produce output
* [x] pr-validation throws an error around ClassLoaderLayeringStrategy
* [ ] https://github.com/jrudolph/sbt-dependency-graph/issues/178?
* [x] release of sbt 1.3.0 itself

### CODE REVIEW

**Comment by raboof** [comment]
Apart from https://github.com/lightbend/sbt-whitesource/issues/58, 'sbt paradox' no longer appears to generate an `index.html`

**Comment by raboof** [comment]
Also should unblock #25830

**Comment by raboof** [comment]
PLS BUILD

**Review by helena — APPROVED** [review]
I was really hoping to do this upgrade, so glad to see it. LGTM!

**Comment by helena** [comment]
Let's not wait for  jrudolph/sbt-dependency-graph#178?

**Comment by patriknw** [comment]
Have we tried release dry-run (unidoc etc)?

**Comment by raboof** [comment]
Hold on, there's still a whitesource problem (looks like https://github.com/lightbend/sbt-whitesource/pull/68)

**Comment by raboof** [comment]
Failure was #27757

**Comment by raboof** [comment]
PLS BUILD

**Comment by raboof** [comment]
failure was https://github.com/akka/akka/issues/24580

**Comment by raboof** [comment]
PLS BUILD

File: .sbtopts [hunk 1]
```diff
@@ -1,3 +1,2 @@
 -J-Xmx3072M
 -J-Xms1024M
--J-XX:MaxMetaspaceSize=512M
```
Comment (patriknw) [inline .sbtopts]: did something change in sbt, or just general cleanup?
(we have been struggling with full meta space GC pauses on Jenkins, but I'm not sure that was the reason for setting this)

**Review by patriknw — APPROVED** [review]
LGTM

File: .sbtopts (hunk 1 above)
Comment (raboof) [reply .sbtopts]: I think there was a build that ran out... instead of removing it we could also increase it, of course, but that seemed a bit arbitrary

**Pull request merged**